scraper.py

```diff
@@ -796,11 +796,13 @@ def scrape_episodes_from_fireside(shows: Dict[str,ShowDetails] , executor):
         # Use same structure as in the root project for easy copy over
         output_dir = os.path.join(
             DATA_ROOT_DIR, "content", "show", show_slug)
-
-        api_data = ShowJson(
-            **requests.get( show_config.fireside_url + "/json")
-                .json()
-        )
+        try:
+            api_data = ShowJson(
+                **requests.get( show_config.fireside_url + "/json")
+                    .json()
+            )
+        except Exception as e:
+            logger.exception(f"Failed to retrieve JSON for {show_config.name}.")
```

Comment:
I can't remember off hand, but I think this will return just the exception and not the stack trace. If we think the stack trace is helpful, we should add that argument. I think it would be good to have (especially since we were just talking about how we didn't get much detail about the error in the first place.
```suggestion
            logger.exception(f"Failed to retrieve JSON for {show_config.name}.", stack_info=True, exc_info=True)
```

Reply:
It does:
![image](https://github.com/JupiterBroadcasting/show-scraper/assets/11327907/7f9bc7c1-64b0-46c4-8e4e-0ddd729bd96c)

Reply:
Guess I was wrong then! That seems fine to me